build.sbt

```diff
@@ -31,9 +31,14 @@ ThisBuild / packageDoc / publishArtifact := false
 // Disable publishing of the root project
 publish / skip := true
 
-def commonProject(dir: String): Project = {
+def commonProject(
+  dir: String,
+  projectPrefix: String = "",
+): Project = {
+  val packagePrefix = s"com.rallyhealth${if (projectPrefix.isEmpty) "" else s".$projectPrefix"}"
   Project(dir, file(dir)).settings(
     name := s"vapors-$dir",
+    idePackagePrefix := Some(packagePrefix),
   )
```

Comment:
I wonder about symmetry on this (and maybe you've already tested this)?

Should the package prefix end with a dot or not?  When this translates to paths, are we sure that we're not going to end up with "/some/prefix//and/something/added" or worse "/some/prefixand/something/added"?